pycheribuild/projects/cross/llvm.py

```diff
@@ -762,17 +764,58 @@ def configure(self, **kwargs):
 
     def install(self, **kwargs):
         super().install(**kwargs)
-        # FIXME: this appears to break the cheribsd build, so let's remove it for now...
```

Comment:
This just deletes parts of morello-llvm??

Reply:
Yeah not sure what happened in that commit, well spotted.